tests/test_invenio_i18n_selectors.py

```diff
@@ -156,3 +156,42 @@ def test_get_locale_anonymous_user(app):
 
     with app.test_request_context():
         assert "en" == get_locale()
+
+
+def test_get_locale_forced_locale(app):
+    """Test retrieving locale via forced locale."""
+    from invenio_i18n import force_locale
+
+    languages = [
+        ("da", "Danish"),
+        ("en", "English"),
+        ("es", "Spanish"),
+    ]
+    app.config["I18N_LANGUAGES"] = languages
+
+    app.secret_key = "secret key"
+    login_manager = LoginManager()
+    login_manager.init_app(app)
+    login_manager.login_view = "login"
+    InvenioI18N(app)
+
+    for l in languages:
+        with force_locale(l[0]):
+            assert l[0] == get_locale()
```

Comment:
minor: this is calling "our" `get_locale()`. What happens when actually resolving a translated string using `gettext`? I'm curious because it looks like [`flask_babel.get_locale`](https://github.com/python-babel/flask-babel/blob/69d3340cd0ff52f3e23a47518285a7e6d8f8c640/flask_babel/__init__.py#L257) would actually use [the forced value](https://github.com/python-babel/flask-babel/blob/69d3340cd0ff52f3e23a47518285a7e6d8f8c640/flask_babel/__init__.py#L339), without ever reaching the `locale_selector` we set.

If that is the case, can you try also calling `gettext` inside the `force_local()` call, without the fix in our selector?

Reply:
> Should I just update the test case and rename as you suggested and integrate a proxy (as mentioned [here](https://github.com/inveniosoftware/invenio-notifications/pull/15#discussion_r1246923680)) or just export the `force_locale`?

I'm not 100% on the rename, since it might be used in other modules (either by accident or on purpose...), so we can just open an issue and investigate separately.

Since we're going with "exporting" some common names, I would also do that for `force_locale`... I suspect the `get_locale` export might have been a way to provide an abstraction/compatibility at some point, but not need to dig too deep into it :)

Reply:
Created an issue for renaming